app/views/handlers/submission.py

```diff
@@ -57,5 +67,5 @@ def get_payload(self):
     def _store_submitted_time_and_display_address_in_session(self):
         session_data = self._session_store.session_data
         session_data.display_address = self._metadata.get("display_address")
-        session_data.submitted_time = datetime.utcnow().isoformat()
+        session_data.submitted_time = self.submitted_at.isoformat()
```

Comment:
As discussed, we should look to drop `submitted_time` from session data and use the `submitted_at` from the questionnaire state directly.